terraform/main.tf

```diff
@@ -253,34 +255,35 @@ resource "aws_iam_policy" "c11-railway-tracker-realtime-lambda-policy-tf" {
   name        = "c11-railway-tracker-realtime-lambda-policy-tf"
   description = "Policy to allow scheduler to invoke etl realtime lambda function"
   policy      = jsonencode({
-    Version = "2012-10-17",
+    Version   = "2012-10-17",
     Statement = [
       {
-        Effect = "Allow",
-        Action = "lambda:InvokeFunction",
+        Effect   = "Allow",
+        Action   = "lambda:InvokeFunction",
         Resource = aws_lambda_function.c11-railway-tracker-realtime-etl-lambda-function-tf.arn
       }
     ]
   })
 }
+
 resource "aws_iam_role_policy_attachment" "scheduler_pipeline_realtime_lambda_invoke_policy" {
   role       = aws_iam_role.c11-railway-tracker-realtime-etl-schedule-role-tf.name
   policy_arn = aws_iam_policy.c11-railway-tracker-realtime-lambda-policy-tf.arn
 }
 
 # schedule realtime process 12am everyday
 resource "aws_scheduler_schedule" "c11-railway-tracker-realtime-etl-schedule-tf" {
-  name = "c11-railway-tracker-realtime-etl-schedule-tf"
-  group_name = "default"
-  schedule_expression = "cron(0 0 * * ? *)"
+  name                         = "c11-railway-tracker-realtime-etl-schedule-tf"
+  group_name                   = "default"
+  schedule_expression          = "cron(0 0 * * ? *)"
```

Comment:
Every day at midnight? It might be best to offset it by 1 hour in case the API doesn't update exactly at midnight

Reply:
done